packages/core/src/payment/strategies/stripe-upe/stripe-upe-custom-styles.ts

```diff
@@ -0,0 +1,65 @@
+import { StripeFormMode, StripeUPEAppearanceOptions } from './stripe-upe';
+
+interface Styles {
+    [key: string]: string;
+}
+
+export const getStripeCustomStyles = (
```

Comment:
We generally avoid placing CSS styles in the SDK, and using inline styles limits other developers' ability to change the UI via CSS override. Is it possible to put them into `checkout-js`?

Reply:
I think I agree with @bc-peng here, we should not put styles in SDK. Since it's supposed to be a wrapper for API that powers checkout. Unless we have a good reason for we should have styles in checkout-js.